barretenberg_static_lib/src/composer.rs

```diff
@@ -331,9 +331,7 @@ mod test {
             public_inputs: Assignments(vec![Scalar::one(), 3_i128.into()]),
             result: false,
         };
-        let test_cases = vec![
-            /*case_1,*/ case_2, case_3, /*case_4,*/ case_5, case_6,
-        ];
+        let test_cases = vec![case_1, case_2, case_3, case_4, case_5, case_6];
```

Comment:
I looked into these and something weird happens because they print `Serialized transcript does not contain the required number of bytes` which is a `throw_or_abort` error message, which makes bb abort in native. This is the wrong behavior and I believe we should keep these disabled until we properly solve in bb.

Reply:
This is also the reason they don't work in wasm, because `throw` is an unreachable trap and crashes the entire wasm instance.